ZeeguuAPI/ZeeguuAPITests/ZeeguuAPITests.swift

```diff
@@ -47,6 +65,12 @@ class ZeeguuAPITests: XCTestCase {
 		testLock.wait()
 	}
 	
+	func testZZZLogout() {
```

Comment:
All testcases are sorted alphabetically when tested. Therefore, I added ZZZ to force it being executed last (for the same reason there is testAAALogin, to force it being executed first). This way, for all test cases, there is a login available.